config/config.go

```diff
@@ -0,0 +1,19 @@
+package config
+
+import (
+	"os"
+
+	"github.com/joho/godotenv"
+	"go.uber.org/zap"
+)
+
+/*
+	GetEnvVariable - Return an environment variable, defined in a .env file, given a key.
+*/
+func GetEnvVariable(key string) string {
```

Comment:
As a suggestion, single-line comments should be written with //.

```suggestion
// GetEnvVariable - Return an environment variable, defined in a .env file, given a key.
func GetEnvVariable(key string) string {
```